terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/validation/ValidationTest.java

```diff
@@ -0,0 +1,265 @@
+/*

```

Comment:
- [ ] Is this file for this PR? It looks like test implementations for validation libraries...